webapp/src/component/layout/DashboardPage.tsx

```diff
@@ -54,8 +57,15 @@ export const DashboardPage: FunctionComponent<Props> = ({
 
   const rightPanelWidth = useGlobalContext((c) => c.rightPanelWidth);
 
-  const quickStartGuideOpen = useGlobalContext((c) =>
-    Boolean(c.quickStartGuide.open && c.userInfo)
+  const { quickStartSetOpen } = useGlobalActions();
```

Comment:
I guess the convention is to name setters starting "set"

Reply:
Ok, might be so